sqlglot/parser.py

```diff
@@ -3010,6 +3015,13 @@ def _parse_foreign_key(self) -> exp.Expression:
         )
 
     def _parse_primary_key(self) -> exp.Expression:
+        desc = None
```

Comment:
desc = (self.Match(ASC) or ...) and token_type == TokenType.desc

Reply:
Rewrote it with `self._match_set` instead.